docker/mongoDB/mongo-history-init.js

```diff
@@ -0,0 +1,75 @@
+conn = new Mongo();
+db = conn.getDB("history");
+
+db.getCollection('history-transactions-view').insertMany([{
```

Comment:
here we can add PM because this collection will be dedicated to PM
```suggestion
db.getCollection('pm-transactions-history').insertMany([{
```

Reply:
RIght, solved with [d3893ba](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/138/commits/d3893bad90e4e53ecbe0972b27f54dbff5650808)